modules/ROOT/pages/update-streams-switching.adoc

```diff
@@ -0,0 +1,25 @@
+
+= Switching to a Different Stream
+
+In order to switch between the different streams of Fedora CoreOS
```

Comment:
Nit: spurious newline in the middle of the sentence?

Reply:
fixed